engine/dumps/bank03.asm

```diff
@@ -2270,7 +2270,7 @@ _BillsPC:
 	ret
 
 Texte224:
```

Comment:
Even if you are unsure of the translation, you can still use it as a label for now.

```suggestion
WhatAreYouGoingToDoText:
```